packages/terra-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -392,9 +404,18 @@ function WorklistDataGrid(props) {
     }
   };
 
+  const onFocus = () => {
+    gridReceivedFocus.current = true;
```

Comment:
This is going to trigger a lot.  You don't have the logic to ensure that you are not changing focus between children like the DataGrid component.

Reply:
Then should you remove the event listener after the first trigger?  Is there a reason to maintain the listener afterwards?

Reply:
Done